Apeswap-Bots/detect-large-banana-token-mints/src/agent.ts

```diff
@@ -0,0 +1,57 @@
+import {
+  BlockEvent,
+  Finding,
+  HandleBlock,
+  HandleTransaction,
+  TransactionEvent,
+  FindingSeverity,
+  FindingType,
+} from "forta-agent";
+
+import { formatEther } from "@ethersproject/units";
+
+import { createFinding, contractMetaData, contractType } from "./utils";
+
+import { GLOBALS } from "./constants";
+
+const { BANANA_MINT_FUNCTION, BANANA_MINT_AMOUNT } = GLOBALS;
+export let exportedNetwork: string;
+
+const mintTxHandler = (functionAbi: string | string[], contractInfo: contractType): HandleTransaction => {
```

Comment:
The `functionAbi` should not be passed as a parameter, see [here](https://github.com/NethermindEth/Forta-Agents/pull/208#discussion_r862505013).
Also, the `contractInfo` is unused.